migrations/tests/test_0020_update_mediafile_meeting_id.py

```diff
@@ -11,6 +11,37 @@ def test_migration(write, finalize, assert_model):
             "fields": {"id": 12, "meeting_id": 8},
         },
     )
+    write(
+        {
+            "type": "update",
+            "fqid": "mediafile/12",
+            "fields": {"title": "blablabla"},
+        },
+    )
+    write(
+        {
+            "type": "create",
+            "fqid": "mediafile/13",
+            "fields": {"id": 13, "meeting_id": 8},
+        },
+        {
+            "type": "update",
+            "fqid": "meeting/8",
+            "fields": {"mediafile_ids": [12, 13]},
+        },
+    )
+    write(
+        {
+            "type": "delete",
+            "fqid": "mediafile/13",
+            "fields": {},
+        },
+        {
+            "type": "update",
+            "fqid": "meeting/8",
+            "fields": {"mediafile_ids": [12]},
+        },
+    )
```

Comment:
This is more or less redundant with the other events (besides the `delete`). What I'm still missing is a list update event (see e.g. https://github.com/OpenSlides/openslides-backend/blob/49a608ff81bc22a956bb8f17d6d157c9fe443f75/migrations/tests/test_0018_remove_template_meeting_committee_relation.py#L37) and a delete fields event (by providing `None` as a value for a field)

Reply:
Add a list update and two field delete events for the test. I don't think it is important, as the required mediafile/meeting_id is changed into the mediafile/owner_id.